lib/manager/npm/post-update/npm.ts

```diff
@@ -113,7 +113,7 @@ export async function generateLockFile(
     // massage to shrinkwrap if necessary
     if (
       filename === 'npm-shrinkwrap.json' &&
-      (await pathExists(upath.join(cwd, 'package-lock.json')))
+      (await localPathExists('package-lock.json'))
```

Comment:
We need to check if cwd is repo root or package root. Otherwise this now fails when the files are in a repo subfolder.